frontend/docs/application_config.md

```diff
@@ -502,83 +725,17 @@ Example of this option enabled on tables and dashboards:
   },
 ```
 
+## Search Pagination
+*TODO
 
 ## Table Lineage
-
-_TODO: Please add doc_
+*TODO
```

Comment:
Would love your help @allisonsuarez with this one

Reply:
does this include the Amundsen graph UI or just the up/downstream lineage tabs?

Reply:
I leave it up to you!